backend/api/contacts/src/main/java/co/airy/core/contacts/ContactsController.java

```diff
@@ -63,9 +65,37 @@ public ContactsController(Stores stores) {
     }
 
     @PostMapping("/contacts.import")
-    public ResponseEntity<?> importContacts() {
-        // TODO import an array of contacts
-        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+    public ResponseEntity<?> importContacts(@RequestBody @Valid List<CreateContactPayload> payload) {
+        List<Metadata> contactsMetadata = new ArrayList<Metadata>();
+        List<ContactResponsePayload> createdContacts = new ArrayList<ContactResponsePayload>();
+
+        payload.stream().forEach((p) -> {
+            final Contact newContact = Contact.builder()
+                    .id(UUID.randomUUID().toString())
+                    .createdAt(Instant.now().toEpochMilli())
```

Comment:
Wondering if we should allow users to set the creation timestamp? 🤔  Part of me thinks that it's more valuable to know for sure when airy first saw this contact.

Reply:
Not sure I don't have a strong opinion on this. There are some cases it can be useful to set the creation time. But I agree in most cases it will be set automatically

Reply:
Agreed, then how about: Since we don't allow setting creation time for any other object I'd opt for disallowing it here, too. Until we get a concrete requirement to change this.